draft-linus-trans-gossip-ct.md

```diff
@@ -306,7 +308,7 @@ STHs from them.
 STH Pollination is carried out by sending STHs to HTTPS servers
 supporting the protocol, and retrieving new STHs. In the case of
 HTTPS clients, STHs are sent
-in an already established TLS session. This makes it hard for an
+in an already established TLS session [sounds like you're assuming this is done using keepalive, right? Are servers required to support keepalive?]. This makes it hard for an
```

Comment:
Well... yes. I think we were assuming it would be done using keepalive.  The intent was we didn't want to initiate an out-of-band TLS connection for either SCT feedback or STH pollination, as an attacker who has modeled a client's behavior would know "Okay, there's one connection and then a small lag and then a new connection, and if I kill the second connection, there will be no user feedback and I block the CT reporting mechanism."

So for that reason, I think our preference would be that this be done with KeepAlive or HTTP/2, and failing that you could fall back to the separate TLS connection route.